website/components/LiveExample.tsx

```diff
@@ -0,0 +1,83 @@
+import MenuLiveExample from 'components/live-examples/MenuLiveExample';
+import ModalLiveExample from 'components/live-examples/ModalLiveExample';
+import MongoNavLiveExample from 'components/live-examples/MongoNavLiveExample';
+import PaletteLiveExample from 'components/live-examples/PaletteLiveExample';
+import PipelineLiveExample from 'components/live-examples/PipelineLiveExample';
+import PopoverLiveExample from 'components/live-examples/PopoverLiveExample';
+import PortalLiveExample from 'components/live-examples/PortalLiveExample';
+import RadioBoxGroupLiveExample from 'components/live-examples/RadioBoxGroupLiveExample';
+import RadioGroupLiveExample from 'components/live-examples/RadioGroupLiveExample';
+import SideNavLiveExample from 'components/live-examples/SideNavLiveExample';
+import StepperLiveExample from 'components/live-examples/StepperLiveExample';
+import SyntaxLiveExample from 'components/live-examples/SyntaxLiveExample';
+import TableLiveExample from 'components/live-examples/TableLiveExample';
+import TabsLiveExample from 'components/live-examples/TabsLiveExample';
+import TextAreaLiveExample from 'components/live-examples/TextAreaLiveExample';
+import TextInputLiveExample from 'components/live-examples/TextInputLiveExample';
+import ToastLiveExample from 'components/live-examples/ToastLiveExample';
+import ToggleLiveExample from 'components/live-examples/ToggleLiveExample';
+import TokensLiveExample from 'components/live-examples/TokensLiveExample';
+import TooltipLiveExample from 'components/live-examples/TooltipLiveExample';
+import TypographyLiveExample from 'components/live-examples/TypographyLiveExample';
+import Component from 'utils/component';
+
+const map: Record<Component, React.FC> = {
+  badge: BadgeLiveExample,
+  banner: BannerLiveExample,
+  box: BoxLiveExample,
+  button: ButtonLiveExample,
+  callout: CalloutLiveExample,
+  card: CardLiveExample,
+  checkbox: CheckboxLiveExample,
+  code: CodeLiveExample,
+  ['confirmation-modal']: ConfirmationModalLiveExample,
+  copyable: CopyableLiveExample,
+  ['icon-button']: IconButtonLiveExample,
+  icon: IconLiveExample,
+  ['inline-definition']: InlineDefinitionLiveExample,
+  logo: LogoLiveExample,
+  ['marketing-modal']: MarketingModalLiveExample,
+  menu: MenuLiveExample,
+  modal: ModalLiveExample,
+  ['mongo-nav']: MongoNavLiveExample,
+  palette: PaletteLiveExample,
+  pipeline: PipelineLiveExample,
+  popover: PopoverLiveExample,
+  portal: PortalLiveExample,
+  ['radio-box-group']: RadioBoxGroupLiveExample,
+  ['radio-group']: RadioGroupLiveExample,
+  ['side-nav']: SideNavLiveExample,
+  stepper: StepperLiveExample,
+  syntax: SyntaxLiveExample,
+  table: TableLiveExample,
+  tabs: TabsLiveExample,
+  ['text-area']: TextAreaLiveExample,
+  ['text-input']: TextInputLiveExample,
+  toast: ToastLiveExample,
+  toggle: ToggleLiveExample,
+  tokens: TokensLiveExample,
+  tooltip: TooltipLiveExample,
+  typography: TypographyLiveExample,
+};
```

Comment:
I can't help but feel there has to be a better way... Maybe not worth addressing now.

Reply:
For sure a better way, but I cant read from the filesystem in this file. Was talking with @lazytype and I think at some point we should write a script that autogenerates all of the files that are so heavily based upon our fs